zerver/openapi/zulip.yaml

```diff
@@ -1448,6 +1454,14 @@ paths:
                                     Currently always true for public streams (i.e. `"invite_only": false` implies
                                     `"history_public_to_subscribers": true`), but clients should not make that
                                     assumption, as we may change that behavior in the future.
+                                is_support_stream:
+                                  type: boolean
+                                  description: |
+                                    Note: Only present if the changed property was `is_support_stream`.
+                                    
+                                    Whether 
+
+                                    **Changes**: New in Zulip 5.0 (feature level 71).
```

Comment:
How'd you get that version number? Be careful when copy-pasting :P

Reply:
Yeah, I was just creating a template for myself to work on later. This PR wasn't ready for review back then.